### ISSUE TITLE
[A11y] Collection Landing - Link purpose - "i" icon

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1870]
### Operating System

Windows

### Browser

Edge

### Affected page

Collection Show

### Admin user?

Yes

### Issue Description

The "i" next to "Oregon State University" links to LC's page about the term, but for some reason its accessible name is "Open link in new window".  This one is especially confusing because an aria label is used to override the attempt at making that link text be "learn more", but that also isn't a meaningful piece of text.

### Suggested Fix

The "i" icons need some kind of explanation about where they take you, and even if "learn more" were the accessible name, that would be iffy.  In most cases the link text can be the destination page's title, so in this case it could be "Oregon State University - LC Linked Data Service...".  But this icon is probably a pretty general thing we use, so a more generic term might be good like "learn more about this taxonomy term" or something.  Dunno.  This should probably get more experts in the discussion.

### Can anybody QA or does this need an expert?

Needs expert

### QA Steps

Verify that the accessible name of the icon is meaningful within the context of the item.  Exact verbiage requires discussion, and I'll update this section once we figure that out, but testing that it's at least better than "Open link in new window" should be pretty easy :)

**Comment by jechols** [comment]
Sent this question to the webaim list: https://webaim.org/discussion/mail_thread?thread=10202

**Comment by CGillen** [comment]
This is a duplicate of #1750
